boards/BMS/Inc/App/configs/App_PreChargeThreshold.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define MIN_PRE_CHARGE_DURATION_MS (1110U)
```

Comment:
Good question. The precharge RC time constant is 480ms. If you solve for the time it takes for the cap to charge to 90% (rounded to the nearest 10ms) this will be the value in ms